Mods/SML/Source/SML/Private/Toolkit/KismetBytecodeDisassembler.cpp

```diff
@@ -1099,26 +1108,6 @@ bool FSMLKismetBytecodeDisassembler::FindFirstStatementOfType(UStruct* Function,
 	return false;
 }
 
-bool FSMLKismetBytecodeDisassembler::GetStatementLength(UStruct* Function, int32 ExpectedStatementIndex, int32& OutStatementLength) {
```

Comment:
This is part of the API, I would not remove it. Not in this commit, at least.

Reply:
Added back.

Reply:
Ack